ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Embed/Nary.hs

```diff
@@ -125,7 +125,7 @@ instance Inject SerialisedHeader where
 instance Inject WrapHeaderHash where
   inject _ (idx :: Index xs x) =
     case dictIndexAll (Proxy @SingleEraBlock) idx of
-      Dict ->
+      Dict.Dict ->
```

Comment:
Any particular reason to qualify `Dict` here? In other places, you imported it unqualified (but still explicitly).

Reply:
No reason honestly, let me fix that